compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.td

```diff
@@ -591,6 +591,19 @@ def AMDAIE_NpuHalfDmaCpyNdOp
     ShapedType::kDynamic encodes that the corresponding entry has a dynamic
     value.
 
+    It also supports the representation of DMA BD chaining using the `use_next_bd`, 
+    `next_bd`, and `start_bd` operands. The `use_next_bd` operand indicates 
+    whether another DMA operation is chained to follow this one. 
+    If `use_next_bd` is `true`, the `next_bd` operand specifies the BD ID of 
+    the next DMA operation in the chain. 
+    
+    The `start_bd` operand specifies the BD ID of the first DMA operation in a sequence.
+    - If `start_bd` is the same as `bd_id`, it marks the start of a chain.
+    - If `start_bd` differs from `bd_id` and `use_next_bd` is `true`, it represents 
```

Comment:
Having a `start_bd` in every op of the chain can easily lead to issues if some transformation would want to update BD IDs. I know why you're doing this, but maybe there are other solutions?

Reply:
Yeah, I agree it might actually be useful to keep the `start_bd` of the chain around.